src/cmd/tools/m3db_tool/main/database/cmd.go

```diff
@@ -0,0 +1,65 @@
+package database
+
+import (
+	"flag"
+	"fmt"
+	"go.uber.org/zap"
+	"os"
+
+	"github.com/m3db/m3/src/cmd/tools/m3db_tool/main/common"
+	"github.com/m3db/m3/src/cmd/tools/m3db_tool/main/flagvar"
+	"github.com/m3db/m3/src/query/generated/proto/admin"
+)
+
+var (
+	CmdFlags   *flag.FlagSet
+	createYaml = flagvar.File{}
```

Comment:
You are not supposed to mix case on acronyms in Go:
~CmdFlags~ - use `CMDFlags` or `cmdFlags` instead; and same for
~ createYaml~ - use `createYAML`